src/ContextMenu.tsx

```diff
@@ -87,6 +87,11 @@ const menuStyles = () => {
       ':focus': {
         outline: '0 none',
       },
+
+      '.react-contextmenu-submenu': {
+        '.react-contextmenu': { opacity: 0, display: 'none' },
```

Comment:
Do we need both `opacity` and `display`?

Reply:
Yeah, display is enough in this case.